docsets/Vulkan/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "Vulkan",
-    "version": "1.0.38",
+    "version": "1.0.39",
     "archive": "Vulkan.tgz",
     "author": {
         "name": "Lei Zhang",
         "link": "https://github.com/antiagainst"
     },
-    "aliases": ["vk", "GLNext", "Graphics API"]
+    "aliases": ["vk", "GLNext", "Graphics API"],
+    "specific_versions": [
+        {
+            "version": "1.0.39",
+            "archive": "versions/1.0.39/Vulkan.tgz"
+        },
+        {
+            "version": "1.0.38",
+            "archive": "versions/1.0.38/Vulkan.tgz"
```

Comment:
Well, it is already removed from the repo and now I'm adding it back again. Sounds a little bit strange, especially we have a new version now and the old version won't be used.

And just to clarify, in the future, say if I want to update 1.0.100, I need to check in all the tarballs for 1.0.38 way up to 1.0.100?